frontend/src/HelperFunctions/validateAndReturnUrl.js

```diff
@@ -0,0 +1,7 @@
+export default function validateAndReturnUrl(url) {
+	var pattern = /^((http|https):\/\/)/;
```

Comment:
Never use var, always let/const :)

Reply:
That's a StackOverflow direct copy and paste mistake of mine :p